packages/quantic/cypress/integration/facets/numeric-facet/numeric-facet-actions.ts

```diff
@@ -18,6 +18,9 @@ const numericFacetActions = (selector: AllFacetSelectors) => {
     submitManualRange: () => {
       selector.searchForm().submit();
     },
+    findCollapseButtonPressTabPressSpace: () => {
```

Comment:
It is quit tricky to find names for these actions. This action intents to

set focus to Collapse button at the top of each facet
press Tab key (expect focus to be set on first selectable row)
Press Enter (expect first selectable row to be selected using Enter key)

Reply:
So, if I understand correctly, it selects the first value of each facet?

Reply:
Yes it selects the first value of the numeric-facet

Reply:
I guess `selectFirstValue` would better carry the meaning then. Now, to make it clearer that it is meant for accessibility, we have a couple options:

* append `UsingKeyboard` to the function name so it becomes `selectFirstValueUsingKeyboard`
* prefix with `a11y` to make it obvious that it is meant to test accessibility: `a11ySelectFirstValue`
* or use a separate object to store the accessibility actions. The function call would then look like `A11yNumericFacetActions.selectFirstValue()`

Since we are expecting to add many actions to test accessibility, I think the third option would be the cleanest.

Reply:
`selectFirstValueUsingKeyboard` used for this action.